frontend/src/pages/Home/Home.tsx

```diff
@@ -6,12 +6,19 @@ import { useNavigate } from 'react-router';
 
 import { useConfig } from '../../hooks';
 import { useEffect, useState } from 'react';
+import { NavigationMenu, type NavItem } from '../../components/NavigationMenu';
 
 interface User {
   name: string;
   email: string;
 }
-
+const items: NavItem[] = [
```

Comment:
it would be nice to create some `routePaths` config and move routes to the separate file, like:
```
export const routePaths = {
  home: () => '/',
  myList: () => '/list',
  ...
  }
```